src/components/Section/__snapshots__/Section.test.jsx.snap

```diff
@@ -4813,243 +4813,61 @@ exports[`The section component renders financial.taxes 1`] = `
 `;
 
 exports[`The section component renders foreign.business 1`] = `
-<div>
+<div
+  className="section-view"
+>
   <div
-    className="section-view"
+    className="view view-business"
   >
     <div
-      className="view view-foreign"
+      className="bottom-btns"
     >
-      <div>
-        <div>
-          <div>
+      <div
+        className="btn-wrap"
+      >
+        <div
+          className="btn-container"
+        >
+          <div
+            className="btn-cell"
+          />
+          <div
+            className="btn-spacer"
+          />
+          <a
+            aria-label="Go to next section Identification intro"
```

Comment:
This is a great find!

The `Section` snapshot tests are a little dodgy since introducing the the next section navigation back/next. The new back/next buttons are leveraging the new config, meanwhile the `Section` snapshots are leveraging the old `navigation.js` config. I'm not sure whether to fix this here or after all sections have been refactored or after `navigation.js` is deprecated.

Reply:
I can't update the `Section` snapshot tests to leverage the new config because some of the sections that haven't been refactored yet will break the tests. I've noticed `SubstanceUse` break the tests.

My impression is that once we update it to use the new config, the existing snapshots will be obsolete and need to get updated.